homeassistant/components/camera/skybell.py

```diff
@@ -19,26 +24,48 @@
 
 SCAN_INTERVAL = timedelta(seconds=90)
 
+IMAGE_AVATAR = 'avatar'
+IMAGE_ACTIVITY = 'activity'
+
+CONF_ACTIVITY_NAME = 'activity_name'
+CONF_AVATAR_NAME = 'avatar_name'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_MONITORED_CONDITIONS, default=[IMAGE_AVATAR]):
+        vol.All(cv.ensure_list, [vol.In([IMAGE_AVATAR, IMAGE_ACTIVITY])]),
+    vol.Optional(CONF_ACTIVITY_NAME, default=""): cv.string,
+    vol.Optional(CONF_AVATAR_NAME, default=""): cv.string,
+})
+
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the platform for a Skybell device."""
+    cond = config[CONF_MONITORED_CONDITIONS]
+    names = {}
+    names[IMAGE_ACTIVITY] = config[CONF_ACTIVITY_NAME]
+    names[IMAGE_AVATAR] = config[CONF_AVATAR_NAME]
     skybell = hass.data.get(SKYBELL_DOMAIN)
 
     sensors = []
     for device in skybell.get_devices():
-        sensors.append(SkybellCamera(device))
+        for camera_type in cond:
+            sensors.append(SkybellCamera(device, type, names[camera_type]))
```

Comment:
Use a different variable name than `type`. That's a reserved word (builtin) in python.

Reply:
Oh thanks for catching this I actually changed this to camera_type and missed this one.